### PR TITLE
Fixes on top of PR-197 during e2e testing

### PR DESCRIPTION
**Pull request by ShyamsundarR**
Submitting as a reference for others...

### CODE REVIEW

**Review by BenamarMk — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -123,7 +123,7 @@ run-hub: generate manifests ## Run DR Orchestrator controller from your host.
 run-dr-cluster: generate manifests ## Run DR manager controller from your host.
 	go run ./main.go --config=examples/dr_cluster_config.yaml
 
-docker-build: test ## Build docker image with the manager.
+docker-build: ## Build docker image with the manager.
```
Comment (vdeenadh) [inline Makefile:126]: Is this change related to the PR subject?

**Review by vdeenadh — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (ShyamsundarR) [reply Makefile:126]: No no... :) i just commented out test as I was merging and wanted to finish e2e, so this would need to be reverted if we are to merge this. More later...

**Comment by ShyamsundarR** [comment]
Fixes absorbed into #197 closing this PR.
